otter/test/convergence/test_service.py

```diff
@@ -790,7 +842,7 @@ def plan(*args, **kwargs):
                                status=ScalingGroupStatus.ERROR),
              noop),
             (Log('group-status-error', dict(isError=True, cloud_feed=True,
-                                            status='ERROR')),
+                                            status='ERROR', reasons='')),
```

Comment:
Huh, I thought I had a test with reasons, I must've forgot! Thanks for pointing that out :)